02-peer-protocol.md

```diff
@@ -580,31 +598,51 @@ The receiving node:
   - if `fee_satoshis` is equal to its previously sent `fee_satoshis`:
     - SHOULD sign and broadcast the final closing transaction.
     - MAY close the connection.
-  - otherwise, if `fee_satoshis` is greater than
-the base fee of the final commitment transaction as calculated in
-[BOLT #3](03-transactions.md#fee-calculation):
-    - MUST fail the connection.
-  - if `fee_satoshis` is not strictly
-between its last-sent `fee_satoshis` and its previously-received
-`fee_satoshis`, UNLESS it has since reconnected:
+  - if `fee_satoshis` matches its previously sent `fee_range`:
+    - SHOULD use `fee_satoshis` to sign and broadcast the final closing transaction
+    - SHOULD reply with a `closing_signed` with the same `fee_satoshis` value if it is different from its previously sent `fee_satoshis`
+    - MAY close the connection.
+  - if the message contains a `fee_range`:
+    - if there is no overlap between that and its own `fee_range`:
```

Comment:
Warnings don't exit yet though right? Also this says nothing w.r.t what should be in the warning message itself, so the message would be sent w/o any context.

Reply:
Correct, see https://github.com/lightningnetwork/lightning-rfc/pull/847#issuecomment-899832302. As for warning messages being more clearly defined, that can come in a followup to the warning message pr.

Reply:
Done in https://github.com/lightningnetwork/lightning-rfc/pull/847/commits/92d2af04069ad5f3668bc3605353243a463c6169